README.md

```diff
@@ -1,40 +1,156 @@
 [![Stable](https://img.shields.io/badge/status-stable-brightgreen?style=for-the-badge)](https://github.com/kubewarden/community/blob/main/REPOSITORIES.md#stable)
 
-Please, note well: this file and the scaffold were generated from [a
-template](https://github.com/kubewarden/rust-policy-template). Make
-this project yours!
+# High-Risk Service Account Blocker
 
-You can use `cargo generate -g https://github.com/kubewarden/rust-policy-template.git`
-to create your Policy from this template.
+Kubernetes [service
+accounts](https://kubernetes.io/docs/concepts/security/service-accounts/) are
+account types used by workloads running in the cluster. A service account can
+grant credentials to workloads, enabling them to perform a wide range of
+cluster operations. This can be dangerous when a workload is able to manipulate
+resources it is not authorized to. This policy aims to mitigate such risks by
+preventing resources that utilize high-risk service accounts from being
+deployed in the cluster. To achieve this, the policy inspects the roles
+associated with the service account in use. If any of the rules are considered
+high-risk, the request is rejected.
 
-# Kubewarden policy high-risk-service-account-policy
+Every time a resource that defines a service account is submitted to the
+cluster, the policy will fetch its associated roles and cluster roles and check
```

Comment:
I would prefer something like:

The policy will query the Kubernetes authorization API the to check if the given ServiceAccount has some permissions that it shouldn't.